packages/postcss/mixin.core.js

```diff
@@ -131,7 +138,15 @@ class PostCSSMixin extends Mixin {
     allLoaderConfigs.splice(
       allLoaderConfigs.indexOf(jsLoaderConfig),
       0,
-      getCSSLoaderConfig(this.config.browsers, require.resolve('style-loader'))
+      getCSSLoaderConfig(this.config.browsers, {
+        loader: require.resolve('style-loader'),
```

Comment:
Like the `mini-css-extract-plugin`'s loader, the `style-loader` now also needs a little configuration to work as expected.